portability-gateway/src/main/java/org/dataportabilityproject/gateway/PortabilityAuthServiceProviderModule.java

```diff
@@ -0,0 +1,45 @@
+package org.dataportabilityproject.gateway;
+
+import com.google.common.collect.ImmutableList;
+import com.google.inject.AbstractModule;
+import com.google.inject.multibindings.MapBinder;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.ServiceLoader;
+import org.dataportabilityproject.spi.gateway.auth.AuthServiceProvider;
+import org.dataportabilityproject.spi.gateway.auth.AuthServiceProviderRegistry;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class PortabilityAuthServiceProviderModule extends AbstractModule {
+
+  private static final Logger logger =
+      LoggerFactory.getLogger(PortabilityAuthServiceProviderModule.class);
+
+  private final ImmutableList<String> enabledServices;
+
+  PortabilityAuthServiceProviderModule(List<String> enabledServices) {
```

Comment:
Maybe this should be a Set unless we care about order.

Reply:
Done